Author: David Mohammed <fossfreedom@ubuntu.com>
Last-Update: 2019-07-11
Description: [PATCH] Resolve lintian issues due to exe install locations (#136)
Origin: Commit 3f78451bab9f152e6c2f1d4133316afa8e086fd3
---
 data/Makefile.am                          | 5 ++++-
 data/slick-greeter-set-keyboard-layout.1  | 7 +++++++
 data/{slick-greeter.1 => slick-greeter.8} | 2 +-
 3 files changed, 12 insertions(+), 2 deletions(-)
 create mode 100644 data/slick-greeter-set-keyboard-layout.1
 rename data/{slick-greeter.1 => slick-greeter.8} (97%)

diff --git a/data/Makefile.am b/data/Makefile.am
index b199bd6..98bfbdc 100644
--- a/data/Makefile.am
+++ b/data/Makefile.am
@@ -36,9 +36,12 @@ dist_pkgdata_DATA = \
 gsettings_SCHEMAS = x.dm.slick-greeter.gschema.xml
 
 dist_man1_MANS = \
-	slick-greeter.1 \
+	slick-greeter-set-keyboard-layout.1 \
 	slick-greeter-check-hidpi.1
 
+dist_man8_MANS = \
+	slick-greeter.8
+
 EXTRA_DIST = \
 	$(gsettings_SCHEMAS)
 
diff --git a/data/slick-greeter-set-keyboard-layout.1 b/data/slick-greeter-set-keyboard-layout.1
new file mode 100644
index 0000000..479cbd6
--- /dev/null
+++ b/data/slick-greeter-set-keyboard-layout.1
@@ -0,0 +1,7 @@
+.TH SLICK-GREETER-SET-KEYBOARD-LAYOUT 1 2019-07-10 Linux "User commands"
+.SH NAME
+slick-greeter-set-keyboard-layout \- switch keyboard layout
+.SH DESCRIPTION
+.B internal executable to switch the keyboard layout
+.PP
+.SH OPTIONS
diff --git a/data/slick-greeter.1 b/data/slick-greeter.8
similarity index 97%
rename from data/slick-greeter.1
rename to data/slick-greeter.8
index 3631fdc..ffd6cb9 100644
--- a/data/slick-greeter.1
+++ b/data/slick-greeter.8
@@ -1,4 +1,4 @@
-.TH SLICK-GREETER 1 "June 2, 2017"
+.TH SLICK-GREETER 8 2019-07-10 Linux "System management commands"
 .SH NAME
 slick-greeter \- distro agnostic LightDM greeter
 .SH SYNOPSIS
-- 
2.17.1

